index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+
+
+<html lang="en-GB">
+
+<head>
+	<title>	
+		Paul English
+	</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" rel="stylesheet" type="text/css">
+	<link href="styles.css" rel="stylesheet" type="text/css">	
+</head>
+
+
+
+<body>
+	<div class="container">	
+		
+		
+		<nav>
+			<div>
+				<div class="navigation navBorder">
+					<a class="x" href="#AboutMe">About Me </a>
+				</div>
+				<div class="navigation navBorder">
+					<a href="#Portfolio">Portfolio </a>
+				</div>
+				<div class="navigation navBorder">
+					<a href="#ContactMe">Contact Me </a>
+				</div>
+			</div>
+		</nav>
+	
+		<header>
+			<div>	
+				<h1> Paul English (alias Zedonk or Zedonk1995) </h1>	
+			</div>
+		</header>
+		
+		<div class="titleBackgroundFooter">
+			<footer>
+				Photo by Dries Augustyns on Unsplash
+				<!-- titleBackground source: Photo by blurrystock on Unsplash -->
+				<!-- titleBackground2 source: Photo by Dries Augustyns on Unsplash -->
+			</footer>
+		</div>
+	
+	
+	<section class="aboutMe">
+		<h2 id="AboutMe"> About me: </h2>
+		<p>
+			I am an aspiring software developer who finished his MMath from the university of Oxford in the year 2020.  
+			In my spare time I like to make maps for the doom and hexen games and apply my knowledge of ACS and DECORATE 
+			to make modifications to the enemies and levels.  (If you want to see some of the maps I've made, take a look 
+			at the <a href="https://doomwiki.org/wiki/Zedonk">doom wiki.</a>)  I've also made a Unity game.  
+		</p>
+	</section>
+	
+	
+		<main>
+			<section class="projectsSection">
```

Comment:
extra layer of section is not needed. Place the projectsSection on the main tag and remove